pontoon/base/models.py

```diff
@@ -192,6 +204,60 @@ def latest_activity(self, project=None):
         return translations.latest_activity()
 
 
+@receiver(pre_save, sender=Locale)
+def create_locale_translators_group(sender, instance,  raw, **kwargs):
+    """
+    Creates respective translators group in django's permission system.
+    """
+    if raw:
+        return
+
+    locale_ct = ContentType.objects.get(app_label='base', model='locale')
+    locale_group, _ = Group.objects.get_or_create(name='{} translators'.format(instance.code.upper()))
+    can_translate = Permission.objects.get(content_type=locale_ct, codename='can_translate')
+    locale_group.permissions.add(can_translate)
+    instance.translators_group = locale_group
+
+
+@receiver(pre_save, sender=Locale)
+def create_locale_managers_group(sender, instance, raw, **kwargs):
```

Comment:
It would be nice to have a comment here explaining what the function does, similarly as you did with create_locale_translators_group().